connectors/powerbi/PowerBI_Delta.Dataset/model.bim

```diff
@@ -349,6 +349,8 @@
           "",
           "        Delimiter = if Text.Contains(DeltaTableFolderContent{0}[Folder Path], \"//\") then \"/\" else \"\\\",",
           "",
+          "        fn_URIDecode = (data as text) as text => Uri.Parts(\"http://powerbi?txt=\" & data)[Query][txt],",
```

Comment:
Are there any concerns that the Uri.Parts would included `https` not just `http://powerbi`?

Reply:
thats just a random name, could also be `https://abc` or `abc://xyz` - we just need something to generate a valid URI to which we pass the encoded string and then we use `Uri.Parts` to extract the all query parameters (`[Query]`) and then select the one we have set to our encoded string (`[txt]`)

Reply:
Got it!  Thanks!